modules/openapi-generator/src/main/resources/csharp-netcore/ApiResponse.mustache

```diff
@@ -105,22 +115,25 @@ namespace {{packageName}}.Client
         /// <param name="statusCode">HTTP status code.</param>
         /// <param name="headers">HTTP headers.</param>
         /// <param name="data">Data (parsed HTTP body)</param>
-        public ApiResponse(HttpStatusCode statusCode, Multimap<string, string> headers, T data)
+        /// <param name="rawContent">Raw content.</param>
+        public ApiResponse(HttpStatusCode statusCode, Multimap<string, string> headers, T data, string rawContent)
```

Comment:
These would probably be overloaded constructors to avoid breaking existing code (e.g. tests with stubs).

Reply:
Added few overloads for binary compatibility.